### PR TITLE
[FIX] pos_gift_card: Setting customer reset price

### PR DESCRIPTION
**Pull request by robinengels**
Current behavior:
When creating a gift card in a PoS the price was reset to 0 when modifying the customer

Steps to reproduce:
- Set PoS gift card mode in Generate a new barcode and set a price
- Modify the current customer
- The price of the gift card is reset to 0

opw-2666917
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/pos_gift_card/static/src/js/GiftCardPopup.js [hunk 1]
```diff
@@ -47,6 +47,7 @@ odoo.define("pos_gift_card.GiftCardPopup", function (require) {
         quantity: 1,
         merge: false,
         generated_gift_card_ids: giftCard ? giftCard.id : false,
+        extras: { price_manually_set: true }
```
Comment (kitan191) [inline addons/pos_gift_card/static/src/js/GiftCardPopup.js:50]: The fix looks good to me.

A comma is missing at the end of the line.

File: addons/pos_gift_card/static/src/js/GiftCardPopup.js [hunk 2]
```diff
@@ -65,7 +66,7 @@ odoo.define("pos_gift_card.GiftCardPopup", function (require) {
             fields: ["code", "initial_amount", "balance"],
           });
           if (giftCard.length) {
-            this.env.pos.giftCard.push(giftCard[0])
+            this.env.pos.giftCard.push(giftCard[0]);
```
Comment (kitan191) [inline addons/pos_gift_card/static/src/js/GiftCardPopup.js:69]: Don't add a missing semi-colon on a line that is not part of your fix.
It will add unnecessary diff.

**Comment by robinengels** [comment]
@pimodoo , if there is someone else I can ping for PoS related PR just tell me. I feel like I'm spamming you

**Comment by pimodoo** [comment]
you probably should rebase this PR as it seems content merged in this one is not in your diff

**Comment by pimodoo** [comment]
https://github.com/odoo/odoo/pull/82951/files

**Comment by pimodoo** [comment]
@robodoo r+ rebase-ff
